ilc/server/tailor/fragment-hooks.js

```diff
@@ -46,8 +46,8 @@ function insertStart(logger, stream, attributes, headers) {
                     isAsync
                         ? `<!-- Async fragments are not fully implemented yet: ${uri} -->`
                         : id
-                        ? asyncStylesLoadTemplate(uri, id)
-                        : '',
+                          ? asyncStylesLoadTemplate(uri, id)
+                          : '',
```

Comment:
I think our auto-formatting tools differ. I use Prettier, and it does not add the indents. Which tool is considered a standard in this codebase?

Reply:
In this repo prettier is used. But please check if you have the latest node_modules, there was a recent update there that affected formatting.

Reply:
indeed